src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs

```diff
@@ -96,8 +94,7 @@ public SceneView() : this(Interop.SceneView.SceneNew(), true)
         /// Copy constructor.
         /// </summary>
         /// <param name="sceneView">Handle to an object.</param>
```

Comment:
handles are used internally, in the API doc it will be better to say "The source object" / "The reference to a copied object."

Reply:
I changed it.